conf/modules/align_reads.config

```diff
@@ -30,9 +30,21 @@ process {
 
     withName: '.*:ALIGN_READS:MINIMAP2_ALIGN_UNSPLIT' {
         if(params.preset == 'revio' | params.preset == 'pacbio') {
-            ext.args = "-y -x map-hifi --secondary=no -Y"
-        } else if(params.preset == 'ONT_R9' | params.preset == 'ONT_R10') {
-            ext.args = "-y -x map-ont --secondary=no -Y"
+            ext.args = { [
+                "-y",
+                "-x map-hifi",
+                "--secondary=no",
+                "-Y",
+                "-R @RG\\\\tID:${meta.id}\\\\tSM:${meta.id}"
+            ].join(' ') }
+        } else if(params.preset == 'ONT_R10') {
+            ext.args = { [
+                "-y",
+                "-x map-ont",
+                "--secondary=no",
+                "-Y",
+                "-R @RG\\\\tID:${meta.id}\\\\tSM:${meta.id}"
+            ].join(' ') }
```

Comment:
To me, this does not seem to relate the to sex check, so I would add a line to the changelog about it